packages/core/test/card/cardTests.tsx

```diff
@@ -5,40 +5,31 @@
  * and https://github.com/palantir/blueprint/blob/master/PATENTS
  */
 
-import { assert, expect } from "chai";
+import { assert } from "chai";
 import { shallow } from "enzyme";
 import * as React from "react";
 
 import { Card, Classes } from "../../src/index";
 
 describe("<Card>", () => {
-    const noClass = (className: string) => `Card does not have "${className}" class`;
-
     it("supports elevation, interactive, and className props", () => {
         const wrapper = shallow(<Card elevation={3} interactive={true} className={Classes.TEXT_MUTED} />);
 
-        assert.isTrue(wrapper.hasClass(Classes.CARD), noClass(Classes.CARD));
-        assert.isTrue(wrapper.hasClass(Classes.ELEVATION_3), noClass(Classes.ELEVATION_3));
-        assert.isTrue(wrapper.hasClass(Classes.INTERACTIVE), noClass(Classes.INTERACTIVE));
-        assert.isTrue(wrapper.hasClass(Classes.TEXT_MUTED), noClass(Classes.TEXT_MUTED));
+        assert.isTrue(wrapper.hasClass(Classes.CARD));
+        assert.isTrue(wrapper.hasClass(Classes.ELEVATION_3));
+        assert.isTrue(wrapper.hasClass(Classes.INTERACTIVE));
+        assert.isTrue(wrapper.hasClass(Classes.TEXT_MUTED));
```

Comment:
ok so the msg arg is useful here cuz the default message will be "expected false to be true."
add the second arg with the class name like I suggested a while back:
`assert.isTrue(wrapper.hasClass(Classes.TEXT_MUTED), Classes.TEXT_MUTED);`